### PR TITLE
[Ahub] Fix svace issue @open sesame 04/06 09:40

### PR DESCRIPTION
**Pull request by baek2sm**
Fixed some svace issues on svace-tizen_7.5.

Add exception handling to
- tensor_trainer_nntrainer.cc
- genModelExeOrder.cpp

Signed-off-by: Seungbaek Hong <sb92.hong@samsung.com>

### CODE REVIEW

File: nnstreamer/tensor_trainer/tensor_trainer_nntrainer.cc [hunk 1]
```diff
@@ -532,7 +538,9 @@ void NNTrainer::NNTrainerTrain::createModel() {
 NNTrainer::NNTrainerTrain::NNTrainerTrain(
   const GstTensorTrainerProperties *prop, const std::string &_model_config) :
   num_push_data(0),
-  model_config(_model_config) {
+  model_config(_model_config),
+  training_loss(0),
+  validation_loss(0) {
```
Comment (songgot) [inline nnstreamer/tensor_trainer/tensor_trainer_nntrainer.cc:543]: Could you add it as below?
```
training_loss(0),
validation_loss(0),
num_push_data(0),
model_config(_model_config) {
```

**Comment by songgot** [comment]
Thank you.

**Review by songgot — APPROVED** [review]
LGTM

**Review by DonghakPark — APPROVED** [review]
LGTM!

**Review by jijoongmoon — APPROVED** [review]
LGTM

**Review by jihochu — APPROVED** [review]
LGTM

**Pull request merged**